src/components/book/index.js

```diff
@@ -102,7 +102,6 @@ class Book extends Component {
   }
   render() {
     // eslint-disable-next-line no-console
-    console.log(this.props.location)
```

Comment:
I've been leaving this in for now so that I can verify that the URL is updating as expected. The line before it tells the linter to leave it alone.

We'll either want to leave it or clear out the comment before it as well.